lnd.go

```diff
@@ -755,8 +753,28 @@ func Main(cfg *Config, lisCfg ListenerCfg, shutdownChan <-chan struct{}) error {
 
 // getTLSConfig returns a TLS configuration for the gRPC server and credentials
 // and a proxy destination for the REST reverse proxy.
-func getTLSConfig(cfg *Config) (*tls.Config, *credentials.TransportCredentials,
-	string, func(), error) {
+func getTLSConfig(cfg *Config) ([]grpc.ServerOption, []grpc.DialOption,
+	func(net.Addr) (net.Listener, error), func(), error) {
+
+	// For our REST dial options, we'll increase the max message size that
+	// we'll decode to allow clients to hit endpoints which return more
+	// data such as the DescribeGraph call.  We set this to 200MiB atm.
+	// Should be the same value as maxMsgRecvSize in cmd/lncli/main.go.
+	var serverOpts []grpc.ServerOption
+	restDialOpts := []grpc.DialOption{
+		grpc.WithDefaultCallOptions(
+			grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200),
```

Comment:
nit: could be just `200 * 1024 * 1024`

Reply:
Or if we touch that code anyway, let's make it a constant somewhere that we can also use in `lncli`.